ERCS/erc-7614.md

```diff
@@ -0,0 +1,263 @@
+---
+eip: 7614
```

Comment:
```suggestion
eip: 7616
```
Assigning next sequential EIP/ERC/RIP number.  7614 was already assigned to RIP7614: https://github.com/ethereum/RIPs/pull/10/files
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.

Reply:
Updated. Thanks